piptools/scripts/compile.py

```diff
@@ -33,7 +33,7 @@
 )
 from ..writer import OutputWriter
 
-DEFAULT_REQUIREMENTS_FILE = "requirements.in"
+DEFAULT_REQUIREMENTS_FILES = ("requirements.in", "setup.py", "pyproject.toml")
```

Comment:
```suggestion
DEFAULT_REQUIREMENTS_FILES = ("requirements.in", "setup.py", "setup.cfg", "pyproject.toml")
```

setup.cfg also can be used as a requirement file.